client/json-rpc/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "libra-json-rpc-client"
+version = "0.1.0"
+authors = ["Libra Association <opensource@libra.org>"]
+description = "Libra JSONRPC client"
+repository = "https://github.com/libra/libra"
+homepage = "https://libra.org"
+license = "Apache-2.0"
+publish = false
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Silly question: is this comment really needed? I've never seen this in a .toml file before

Reply:
it is just part of the template that generated by clion, I can remove it.